app/experimenter/static/js/components/RadioButtonInlineLabel.js

```diff
@@ -0,0 +1,120 @@
+import { boundClass } from "autobind-decorator";
+import PropTypes from "prop-types";
+import React from "react";
+import { Form, Row, Col } from "react-bootstrap";
+
+import Error from "experimenter/components/Error";
+import HelpBox from "experimenter/components/HelpBox";
+
+@boundClass
+class RadioButtonInlineLabel extends React.PureComponent {
```

Comment:
Oh good actually I was thinking all our radios should use something like this, maybe we can remove the other radio and just use this for everything.